docs/DeveloperGuide.md

```diff
@@ -379,6 +378,69 @@ is not specified, it would be represented as `"null"`, in the `appointmentbook.j
 
 <br>
 
+## **Appendix: Parsing**
+
+### Design considerations
+
+**Aspect: How to parse the commands:**
+<br>
+Context: The commands (other than the general) have the command format: `add... ENTITY_TYPE ENTITY_ARGS`
```

Comment:
Should we use `COMMAND ENTITY_TYPE ENTITY_ARGS` instead? Since there are other commands other than `add`.

Reply:
Good suggestioN!